setup.py

```diff
@@ -5,26 +5,45 @@
 import Xlib
 
 
-setup(name='python-xlib',
-        version=Xlib.__version_string__,
-
-        description='Python X Library',
-        download_url='https://github.com/python-xlib/python-xlib/releases',
-        url='https://github.com/python-xlib/python-xlib',
-        license='GPL',
-
-        author='Peter Liljenberg',
-        author_email='petli@ctrl-c.liu.se',
-
-        install_requires=['six>=1.10.0'],
-
-        packages=[
-            'Xlib',
-            'Xlib.ext',
-            'Xlib.keysymdef',
-            'Xlib.protocol',
-            'Xlib.support',
-            'Xlib.xobject'
-            ],
-        )
-
+setup(
+    name='python-xlib',
+    version=Xlib.__version_string__,
+
+    description='Python X Library',
+    download_url='https://github.com/python-xlib/python-xlib/releases',
+    url='https://github.com/python-xlib/python-xlib',
+    license='GPL',
```

Comment:
OK, so I'll amend to `LGPLv2+` and the classifier to `License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)`?